src/annotator/anchoring/test/range-js-test.js

```diff
@@ -51,81 +52,89 @@ describe('annotator/anchoring/range-js', () => {
 
   describe('nodeFromXPath', () => {
     let container;
-    let fakeFindChild;
+    const html = `
+        <h1 id="h1-1">text</h1>
+        <p id="p-1">text<br/><br/><a id="a-1">text</a></p>
+        <p id="p-2">text<br/><em id="em-1"><br/>text</em>text</p>
+        <span>
+          <ul>
+            <li id="li-1">text 1</li>
+            <li id="li-2">text 2</li>
+            <li id="li-3">text 3</li>
+          </ul>
+        </span>`;
 
-    beforeEach(() => {
+    beforeEach(function () {
```

Comment:
Can you stick to arrow functions unless a function needs to reference `this`, just for consistency.